lf/system/admin/index.php

```diff
@@ -46,7 +45,7 @@
 
 // Generate new User() and test access
 $user = new User();
-
+$user->fromSession();
 
```

Comment:
I was deving on PHP 5.3 for the longest time. Finally upgraded today. Gonna move everything to that nicer oneliner format ;)